src/components/Help/Help.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
+import { Text } from 'ink';
+import { HelpMessage } from "../../CliModels";
+
+interface HelpInput {
+	message: HelpMessage,
+}
+
+const Help: FC<HelpInput> = ({ message }) => (
+	<Text>{message}</Text>
+);
```

Comment:
In other packages in our organization, where we use React + TS we use the types for FC as `React. FunctionComponent<Props>`, not FC. Not in this PR, but we should unify types :)